### PR TITLE
fix: Update demo tooltip text

### PR DESCRIPTION
**Pull request by weili520**
Signed-off-by: TheYoungManLi <cjl@kubesphere.io>

##[2591](https://github.com/kubesphere/website/issues/2591)

### CODE REVIEW

File: i18n/en.yaml [hunk 1]
```diff
@@ -35,11 +35,15 @@
...
-  translation: "1.Experience account: demo1 / Demo1234"
+  translation: "1.Create a account and login it."
 - id: demo content 2
-  translation: 2.This account is only allowed to view parts of UI
+  translation: 2.Go to kubesphere cloud console page and click managed cluster.
 - id: demo content 3
-  translation: 3.It's recommended that install KubeSphere in your environment
+  translation: 3.Create a kubernetes cluster with Kubesphere and waiting for a while.
+- id: demo content 4
+  translation: 4.Access to the created cluster.
```
Comment (Bettygogo2021) [inline i18n/en.yaml:44]: Access KubeSphere with the default account and password (admin/P@88word).

File: i18n/en.yaml [hunk 2]
```diff
@@ -35,11 +35,15 @@
...
+- id: demo content 5
+  translation: "5.default account: admin/P@88word."
```
Comment (Bettygogo2021) [inline i18n/en.yaml:46]: 这里其实和上一条翻译在一起，建议删除 content 5.

File: i18n/zh.yaml [hunk 3]
```diff
@@ -37,11 +37,15 @@
...
-  translation: "1.体验账号： demo1 / Demo1234"
+  translation: "1.创建账户并登录。"
 - id: demo content 2
-  translation: 2.该账号仅开放了部分 UI 界面查看权限
+  translation: 2.进入管理控制台，点击创建托管集群服务。
 - id: demo content 3
-  translation: 3.建议自行安装体验完整的管理功能
+  translation: 3.创建一个 k8s 托管集群并勾选安装 KubeSphere。
+- id: demo content 4
+  translation: 4.进入托管集群详情页面，点击进入 KubeSphere。
```
Comment (Bettygogo2021) [inline i18n/zh.yaml:46]: 使用默认账户和密码 （admin/P@88word）登录 KubeSphere。

File: i18n/zh.yaml [hunk 4]
```diff
@@ -37,11 +37,15 @@
...
+- id: demo content 5
+  translation: "5.默认账户: admin/P@88word。"
```
Comment (Bettygogo2021) [inline i18n/zh.yaml:48]: 建议删掉

File: i18n/en.yaml [hunk 5]
```diff
@@ -35,11 +35,15 @@
 - id: Demo
   translation: Demo
 - id: demo content 1
-  translation: "1.Experience account: demo1 / Demo1234"
+  translation: "1.Create a account and login it."
```
[the remark this answers is not in the document]
Comment (weili520) [reply i18n/en.yaml]: Done.

File: i18n/en.yaml [hunk 6]
```diff
@@ -35,11 +35,15 @@
...
 - id: demo content 2
-  translation: 2.This account is only allowed to view parts of UI
+  translation: 2.Go to kubesphere cloud console page and click managed cluster.
```
[the remark this answers is not in the document]
Comment (weili520) [reply i18n/en.yaml]: Done.

File: i18n/en.yaml [hunk 7]
```diff
@@ -35,11 +35,15 @@
...
 - id: demo content 3
-  translation: 3.It's recommended that install KubeSphere in your environment
+  translation: 3.Create a kubernetes cluster with Kubesphere and waiting for a while.
```
[the remark this answers is not in the document]
Comment (weili520) [reply i18n/en.yaml]: Done.

File: i18n/en.yaml (hunk 2 above)
Comment (weili520) [reply i18n/en.yaml]: Done.

File: i18n/zh.yaml (hunk 3 above)
Comment (weili520) [reply i18n/zh.yaml]: Done.

File: i18n/zh.yaml (hunk 4 above)
Comment (weili520) [reply i18n/zh.yaml]: Done.

**Review by Arhell — COMMENTED** [review]
/lgtm

**Pull request merged**